### PR TITLE
review structural changes to iRODS documentation

### PR DESCRIPTION
**Pull request by tvandenberghe**
- New article on cadaver/cURL
- New structure

### CODE REVIEW

File: source/data/workflow_automation.rst [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+Rules can be invoked in three ways: by directly calling the ``irule`` command on a rule file (only for User-level rules); by reaching a Policy Enforcement Point, triggering a System-level rule; or periodically via delayed execution (both for User-level and System-level rules). The delay mechanism is used for resource-heavy, time-intensive processes that can occur one or more times in the future at regular intervals.
+
+The Python iRODS Client (PRC) is executed client-side, making it somewhat less efficient than iRODS rule execution. Although both offer overlapping functionality, the delay mechanism used by iRODS rules is more graceful and these rules get stored centrally in the REPF. It is also possible to invoke a rule via the PRC.
+
+This article focuses only on User-level rules. If you have more complex data processing pipelines, the Tier-1 Data Team (FOZ-RDM) at KU Leuven can create System-level rules for you.
```
Comment (jefscheepers) [inline source/data/workflow_automation.rst:16]: Maybe ask here to contact us at data@vscentrum.be?

File: source/data/workflow_automation.rst [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+Rule syntax
+-----------
+
+User-level rules can only be written in the iRODS Rule language, System-level rules can also be written in other languages (Python, C++ or JavaScript). The iRODS Rule language is a domain specific scripting language composed of simple building blocks. It uses curly brackets, # for comments, variable names start with '*' and strings are enclosed by ' or ". The documentation for the language can be found here: https://docs.irods.org/4.2.10/plugins/irods_rule_language/.
```
Comment (jefscheepers) [inline source/data/workflow_automation.rst:21]: @mstfdkmn can user rules also be written in Python?

**Review by jefscheepers — COMMENTED** [review]
Looks good! Only a few comments

File: source/data/workflow_automation.rst (hunk 2 above)
Comment (tvandenberghe) [reply source/data/workflow_automation.rst:21]: Mustafa explained that technically they can but due to a limitation (bug) they are not available for user rules. If or how we express this I leave to you: 'In general they are, but due to technical limitations in our HPC zone they aren't ' or something like this.

**Pull request merged**